icons/roadmapdotsh.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Roadmap.sh</title><path d="M7.7666 22.9874c-.3047.0914-.792.1828-1.462.2741-.6396.0914-1.3097.137-2.0101.137-.6702 0-1.2792-.0456-1.8275-.137-.5179-.0913-.9594-.274-1.3249-.5482-.3654-.2741-.6549-.6396-.868-1.0965C.0914 21.1295 0 20.5052 0 19.7438V6.038c0-.7004.1217-1.2944.3655-1.7817.2741-.5177.6548-.9746 1.1421-1.3706.4874-.396 1.0811-.7461 1.7818-1.0508.731-.335 1.5076-.6244 2.33-.868A17.324 17.324 0 0 1 8.1776.4187a15.5677 15.5677 0 0 1 2.6498-.2285c1.462 0 2.6193.2893 3.4721.868.8527.5483 1.2792 1.462 1.2792 2.7412 0 .4264-.061.8529-.1827 1.2792-.122.396-.2741.7615-.4569 1.0965a21.717 21.717 0 0 0-1.9645.0913 20.2948 20.2948 0 0 0-1.9645.2742c-.6395.1218-1.2488.2589-1.8274.4111-.5482.122-1.0204.2741-1.4162.4569Zm7.3704-3.6548c0-1.2792.3958-2.3452 1.1878-3.198.792-.8529 1.8731-1.2792 3.2437-1.2792s2.4517.4263 3.2437 1.2792c.792.8528 1.1878 1.9188 1.1878 3.198s-.396 2.3451-1.1878 3.198c-.7919.8528-1.8731 1.2792-3.2437 1.2792s-2.4519-.4264-3.2437-1.2792c-.7918-.8529-1.1878-1.9188-1.1878-3.198z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Roadmap.sh</title><path d="M20.693 0H3.307A3.307 3.307 0 0 0 0 3.307v17.386A3.307 3.307 0 0 0 3.307 24h17.386A3.307 3.307 0 0 0 24 20.693V3.307A3.307 3.307 0 0 0 20.693 0zm-7.706 9.18c-.349.031-.689.078-1.021.142-.333.063-.65.134-.95.214a3.64 3.64 0 0 0-.736.237v8.097a5.522 5.522 0 0 1-.76.143c-.333.047-.68.07-1.045.07a5.87 5.87 0 0 1-.95-.07 1.588 1.588 0 0 1-.688-.285 1.476 1.476 0 0 1-.452-.57c-.095-.253-.142-.578-.142-.974V9.061c0-.364.063-.673.19-.926.142-.27.34-.507.594-.713a3.93 3.93 0 0 1 .926-.546 9.133 9.133 0 0 1 2.54-.736 8.093 8.093 0 0 1 1.378-.119c.76 0 1.361.15 1.804.451.444.285.665.76.665 1.425 0 .222-.032.443-.095.665a3.075 3.075 0 0 1-.237.57c-.341 0-.682.016-1.021.047zm5.113 8.453c-.412.443-.974.665-1.686.665s-1.274-.222-1.686-.665c-.412-.443-.617-.998-.617-1.662 0-.665.205-1.22.617-1.663.412-.443.974-.664 1.686-.664s1.274.221 1.686.664c.411.444.617.998.617 1.663 0 .664-.206 1.219-.617 1.662z"/></svg>
```

Reply:
Updated to logo from website header

Reply:
As mentioned in contributing guidelines, the precision could be set to 3 decimal points to be consistent with other SVGs in the project

Reply:
![roadmapdotsh](https://github.com/simple-icons/simple-icons/assets/35000807/ae831bce-4115-46c2-b010-2d9332011233)